go/client/cmd_install_osx.go

```diff
@@ -101,7 +101,72 @@ func (v *CmdInstall) Run() error {
 		fmt.Fprintf(os.Stdout, "%s\n", out)
 	} else {
 		for _, c := range components {
-			v.G().UI.GetDumbOutputUI().Printf("%s: %s %s\n", c.Name, c.Status.Name, c.Status.Desc)
+			v.G().Log.Info("Installer for %s: %s %s", c.Name, c.Status.Name, c.Status.Desc)
+		}
+	}
+	return nil
+}
+
+func NewCmdUninstall(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name: "uninstall",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "o, format",
+				Usage: "Format for output. Specify 'j' for JSON or blank for default.",
+			},
+			cli.StringFlag{
+				Name:  "c, components",
+				Usage: "Components to uninstall, comma separated. Specify 'cli' for command line, 'service' for service, kbfs for 'kbfs', or blank for all.",
+			},
+		},
+		ArgumentHelp: "",
+		Usage:        "Uninstaller",
```

Comment:
can you provide a description to say what it does?  Also, what's it uninstalling?